checks.md

```diff
@@ -510,24 +485,3 @@ spec:
       - name: nginx
         image: nginx:1.7.9
 ```
-
-###### Node Labels and Taints
```

Comment:
This is a recently-added check - looks like it got deleted in a merge/rebase.

Reply:
Re-added and given some love